### DISCUSSION TITLE
Snowflake as Lokinet exit nodes | DHT entry node

### DISCUSSION THREAD

**Discussion by ghost** [oxen-io/lokinet#2239] [Ideas]
- myip>DHT>lokinet node #.loki
- myip>DHT>lokinet node>Snowflake exit  #Clearnet
> Snowflake exit sends traffic to the clearnet.

Just like the principle of Tor Snowflake, to implement Lokinet exit nodes
### Snowflake as Lokinet exit nodes.
- Lokinet exit nodes will increase to several million.
---
- In apps client/browser extensions/OpenWrt plugin/VPS
> Allow users from other countries to use your network as Lokinet exit nodes

## DHT
dht.libtorrent.org:25401
dht.transmissionbt.com:6881
router.bittorrent.com:6881
router.utorrent.com:6881
dht.aelitis.com:6881

https://github.com/holepunchto/hyperdht
https://github.com/libp2p/go-libp2p-kad-dht

**Comment by Ashthetik** [comment]
The Browser plugin would be a difficult situation given how Google is going about extensions, Manifest V3 would most likely play heavily against Loki and would restrict to possibly just a self-contained "API"

I'd be curious to see what your idea for OWRT is, though

**Comment by ghost** [comment]
Approximate survival time of the node.
client `18 hours`
browser extensions `1 minute ~ 3 hours`
OpenWrt extensions /VPS `It's running unless the power goes out.`

**Comment by Ashthetik** [comment]
That...that didn't answer what I asked but alright, I was asking for the technicalities so I could provide unofficial support to either idea whilst Oxen continues to refactor their project
